aws_topology/stackstate_checks/aws_topology/resources/redshift.py

```diff
@@ -8,34 +29,60 @@ class RedshiftCollector(RegisteredResourceCollector):
     COMPONENT_TYPE = "aws.redshift"
     CLOUDFORMATION_TYPE = "AWS::Redshift::Cluster"
```

Comment:
Just for sparring: thinking of getting rid of this one. Mainly because it is strange to have a default CLOUDFORMATION_TYPE for an API while obviously there sometimes two types equally important (like in RDS).

Reply:
Agreed - I've been ignoring it anyway